documentation/staging/content/userguide/managing-domains/accessing-the-domain/port-forward.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Use Port Forwarding"
+date: 2019-02-23T17:39:15-05:00
+draft: false
+weight: 4
+description: "You can use Port Forwarding to access the Administration console and WLST."
+---
+
+
+Beginning with operator version 4.0.0, you can use the `kubectl port-forward` command to allow external access to the WebLogic Administration Console and WLST access to manage a domain running in Kubernetes. You can use this method to access the Administration console locally or connect using WLST for investigating the issues without exposing them to the public internet. See [port-forward](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#port-forward) in the Kubernetes reference documentation for `kubectl port-forward` usage and examples.
+
+The operator automatically adds the required network channels with a 'localhost' address for each existing admin protocol capable port to enable this access. See [Additional network channels created for kubectl port-forward](#additional-network-channels-created-for-kubectl-port-forward) for network channels created by the operator.  You can disable this behavior by setting the `domain.spec.adminServer.adminChannelPortForwardingEnabled` attribute in the domain resource to `false`. Run the `kubectl explain domain.spec.adminServer.adminChannelPortForwardingEnabled` command or see in the domain resource [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md) for more details.
+
+#### Forward a local port to an administration port on the Administration Server Pod
+The `kubectl port-forward` command allows using the resource type/name to select a matching pod to port forward to. You also need to define the local and remote port numbers:
+
+```
+kubectl port-forward TYPE/NAME [options] LOCAL_PORT:REMOTE_PORT
+```
+
+For the WebLogic Administration Console access, you need to forward a local port (on the machine where `port-forward` command is run) to the Administration Server Pod. For example, if you have a WebLogic domain with UID `domain1` running in namespace `mynamespace` and the Administration Server name is `admin-server` with non-SSL listen port 7001, then you can run the below command to forward a local port (port 32015 in this example) to the admin port (port 7001 in this example) of the Administration Server Pod. In this scenario, the Kubernetes API listens on local port 32015 and forwards data to port 7001 on the Administration Server Pod.
+
+```shell
+kubectl port-forward pods/domain1-admin-server -n mynamespace 32015:7001
+```
+or
+
+```shell
+kubectl port-forward service/domain1-admin-server -n mynamespace 32015:7001
+```
+The output is similar to this:
+
+```
+Forwarding from 127.0.0.1:32015 -> 7001
+```
+
```

Comment:
Should there be a cleanup notes for ending the forwarding session?

Reply:
Added a `Terminating the port-forward session` section in 0d1a9a1 .